### ISSUE TITLE
Buttons and Inputs are not flushed when placing together without borders

### ISSUE THREAD

**Issue by Rizzaxc** [themesberg/flowbite-svelte#1166]
## Summary
Buttons and Inputs are slightly misaligned when placing together.
### Basic example

<img width="511" alt="image" src="https://github.com/themesberg/flowbite-svelte/assets/33868883/032182dd-2f9a-4dbd-926d-a48863739c7d">

### Motivation

It's a fairly common design

**Comment by jjagielka** [comment]
Indeed, it is a fairly common design and it's represented in the docs - [Input group](https://flowbite-svelte.com/docs/forms/input-field#Input_group):

```svelte
<ButtonGroup >
<Button color="green">OK</Button><Input placeholder="Input"/>
</ButtonGroup>
```

And I have an impression that elements in such a group are aligned.

**Comment by Rizzaxc** [comment]
> Indeed, it is a fairly common design and it's represented in the docs - [Input group](https://flowbite-svelte.com/docs/forms/input-field#Input_group):
> 
> ```svelte
> <ButtonGroup >
> <Button color="green">OK</Button><Input placeholder="Input"/>
> </ButtonGroup>
> ```
> 
> And I have an impression that elements in such a group are aligned.

I should've worded it better. I had seen the doc prior to posting. In my particular case, I have each element wrapped with its `Label`.
```
<div class="wrapper">
  <div>
    <Label for="myBtn">Btn </Label>
    <Button />
  </div>
  <div>
    <Label for="myInput">Input </Label>
    <Input />
  </div>
</div>
```
What's the correct idiom in here?

**Comment by shinokada** [comment]
You need to use `ButtonGroup` as @jjagielka stated.
